OpenSim/Simulation/Model/ControllerSet.h

```diff
@@ -102,6 +91,13 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ControllerSet, ModelComponentSet<Controller>);
     virtual void computeControls(const SimTK::State& s, SimTK::Vector &controls) const; 
 
     virtual void printInfo() const;
+
+private:
+
+    std::unique_ptr<Storage> _controlStore;
+
+    // Set of actuators controlled by the set of controllers.
+    SimTK::ReferencePtr<Set<Actuator> > _actuatorSet;
```

Comment:
In C++11 the extra space isn't needed any more so you can write `SimTK::ReferencePtr<Set<Actuator>>`.